ipn/ipnlocal/local.go

```diff
@@ -2784,7 +2784,6 @@ func (b *LocalBackend) initPeerAPIListener() {
 	ps := &peerAPIServer{
 		b:                       b,
 		rootDir:                 fileRoot,
-		selfNode:                selfNode,
```

Comment:
remove the selfNode variable above then? It do anything? just keep the Addresses == 0 check

Reply:
its used to calculate the fileRoot.

Reply:
fair nuff